shanoir-ng-datasets/src/main/java/org/shanoir/ng/studycard/dto/StudyCardOnStudyResult.java

```diff
@@ -0,0 +1,80 @@
+package org.shanoir.ng.studycard.dto;
+
+/**
```

Comment:
It's missing the license header

Reply:
I fully agree, should I add it manually or will we add it with your tool?